### PR TITLE
Provide choice to start only rest / grpc or both the services

### PR DESCRIPTION
**Pull request by khansaad**
Signed-off-by: Saad Khan <saakhan@redhat.com>

This PR contains changes to give user a choice a start just the REST service or both at the same time.
Moved the gRPC requirements into a new text file to be called only in case when both service needs to be started.
By default, both services will start. User can provide choice using the newly added flags mentioned in deploy script.

Note: This PR will solve issue #64

### CODE REVIEW

File: deploy_hpo.sh [hunk 1]
```diff
@@ -28,16 +28,19 @@ HPO_CONTAINER_IMAGE=${HPO_REPO}:${HPO_VERSION}
...
 
 # source the helpers script
 . ${SCRIPTS_DIR}/cluster-helpers.sh
 
 function usage() {
 	echo
 	echo "Usage: $0 [-c [docker|minikube|native]] [-o hpo container image]"
-	echo "       -s = start(default), -t = terminate"
+    echo " -s = start(default), -t = terminate"
 	echo " -c: cluster type."
```
Comment (dinogun) [inline deploy_hpo.sh:40]: Almost all the changes have tab / space issues. As mentioned previously, we use tabs for all scripts. I think there is a IDE option to set this locally, easier that way

**Comment by dinogun** [comment]
@please fix the conflicts

**Comment by khansaad** [comment]
> @please fix the conflicts

Done

**Comment by dinogun** [comment]
Conflicts again after #16 was merged, please fix

File: deploy_hpo.sh [hunk 2]
```diff
@@ -33,19 +33,24 @@ non_interactive=0
...
+	echo "Usage:"
+	echo " -a | --non_interactive	: interactive (default)"
+	echo " -s | --start				: start(default) the app"
+	echo " -t | --terminate			: terminate the app"
+	echo " -c | --cluster_type		: cluster type [docker|minikube|native|openshift]]"
+	echo " -o | --container_image	: build with specific hpo container image name [Default - kruize/hpo:<version>]"
+	echo " -n | --namespace			: Namespace to which hpo is deployed [Default - monitoring namespace for cluster type minikube]"
+	echo " -d | --configmaps		: Config maps directory [Default - manifests/configmaps]"
+	echo " -b | --both				: install both REST and the gRPC service"
+	echo " -r | --rest				: install REST only"
```
Comment (dinogun) [inline deploy_hpo.sh:53]: Can you align the options so that they show up consistently on the terminal

Comment (dinogun) [reply deploy_hpo.sh:53]: Can we remove the `-b` and `-r` options and only have `--both` and `--rest` options
Change `--configmaps` option to `--configmaps_dir`

Comment (khansaad) [reply deploy_hpo.sh]: done

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**